src/Scenes/GameScene.cpp

```diff
@@ -316,6 +331,19 @@ void GameScene::saveStatus() {
   GameManager::getInstance()->addMoney(prize_);
 }
 
+std::map<std::string, uint16_t>* GameScene::getPlayerArmy() {
+  std::map<std::string, uint16_t>* army = new std::map<std::string, uint16_t>();
```

Comment:
We already have the type in the same line, no need to specify it twice.
```suggestion
  auto* army = new std::map<std::string, uint16_t>();
```